ui/desktop/electron-app/patches/lodash.template+4.5.0.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/node_modules/lodash.template/index.js b/node_modules/lodash.template/index.js
+index f051141..0c45a8d 100644
+--- a/node_modules/lodash.template/index.js
```

Comment:
are these auto-generated?

Reply:
Yes they are. This patch file is created by the [patch-package](https://www.npmjs.com/package/patch-package) package. If you look at the description one of the steps is `npx patch-package some-package`. This will create the patch for you, after you have altered the source code for that package that requires the patch. Let me know if you have more questions.

Reply:
makes sense, thanks for the clarification